CHANGELOG.md

```diff
@@ -1,7 +1,54 @@
 # CHANGELOG
 
+## v0.32.0
+
+*May 2nd, 2023*
```

Comment:
Was this line manually added? :slightly_smiling_face:

Reply:
Yes,